File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/VehicleprofileAmmo+CoreDataProperties.m
//
//  VehicleprofileAmmo+CoreDataProperties.m
//  WOTData
//
//  Created by Pavel Yeshchyk on 1/17/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

#import "VehicleprofileAmmo+CoreDataProperties.h"

@implementation VehicleprofileAmmo (CoreDataProperties)

+ (NSFetchRequest<VehicleprofileAmmo *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"VehicleprofileAmmo"];
}

@dynamic type;
@dynamic damage;
@dynamic penetration;
@dynamic vehicleprofileAmmoList;

@end

File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/VehicleprofileAmmo+CoreDataProperties.swift
//
//  VehicleprofileAmmo+CoreDataProperties.swift
//
//
//  Created by Pavel Yeshchyk on 4/23/20.
//
//

import Foundation
import CoreData

extension VehicleprofileAmmo {
    @nonobjc public class func fetchRequest() -> NSFetchRequest<VehicleprofileAmmo> {
        return NSFetchRequest<VehicleprofileAmmo>(entityName: "VehicleprofileAmmo")
    }

    @NSManaged public var type: String?
    @NSManaged public var damage: VehicleprofileAmmoDamage?
    @NSManaged public var penetration: VehicleprofileAmmoPenetration?
    @NSManaged public var vehicleprofileAmmoList: VehicleprofileAmmoList?
}
